### ISSUE TITLE
Element is the wrong type for SVG?

### ISSUE THREAD

**Issue by k1sul1** [tanem/svg-injector#351]
I'm using react-svg and the beforeInjection callback doesn't let me access the style property on the svg. [It's because the type is Element. ](https://github.com/tanem/svg-injector/blob/master/src/types.ts#L5)

```
const beforeInject = (svg: Element) => {
  svg.style.backgroundImage = `url(${texture.url})` // TS2339: Property 'style' does not exist on type 'Element'.
  svg.style.width = `${size.width}px`,
  svg.style.clipPath = `url(#${id})`
}
```

It's not really a problem as this works

```
const beforeInject = (s: Element) => {
  const svg = s as HTMLElement

  svg.style.backgroundImage = `url(${texture.url})`
  svg.style.width = `${size.width}px`,
  svg.style.clipPath = `url(#${id})`
}
```

but I was wondering if it's intended or a mistake?

**Comment by tanem** [comment]
Ah, good catch. It should probably be an [`SVGElement`](https://developer.mozilla.org/en-US/docs/Web/API/SVGElement). I'll see what's involved in changing it 👍

**Comment by tanem** [comment]
PR has built successfully, so I'll merge it and create a release for `react-svg`. Will post back here once that's done.

**Comment by tanem** [comment]
Ok, released [`react-svg@11.0.17`](https://www.npmjs.com/package/react-svg/v/11.0.17) if you want to try that out?

**Comment by k1sul1** [comment]
Thanks for fast response! It works. 

This compiles fine: 
```
const beforeInject = (svg: SVGElement) => {
  svg.style.backgroundImage = `url(${texture.url})`
  svg.style.width = `${size.width}px`,
  svg.style.clipPath = `url(#${id})`
}
```
